bluebird.d.ts

```diff
@@ -151,8 +151,8 @@ declare class Bluebird<R> implements Bluebird.Thenable<R>, Bluebird.Inspection<R
    * });
    * </code>
    */
-  // TODO find way to fix get()
-  // get<U>(propertyName: string): Bluebird<U>;
+  // TODO: explain original comment from DT: TODO find way to fix get()
```

Comment:
The point was to use something like subtypes (in TS roadmap I think) to get the type `U` automatically.